tests/load/locustfiles/stored.py

```diff
@@ -0,0 +1,394 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Performance test module."""
+
+import base64
+import json
+import logging
+import random
+import string
+import time
+import uuid
+from hashlib import sha1
+from json import JSONDecodeError
+from logging import Logger
+from typing import Any, TypeAlias
+
+import websocket
+from args import parse_wait_time
+from exceptions import ZeroStatusRequestError
+from locust import FastHttpUser, events, task
+from locust.exception import LocustError
+from models import (
+    HelloMessage,
+    HelloRecord,
+    NotificationMessage,
+    NotificationRecord,
+    RegisterMessage,
+    RegisterRecord,
+    UnregisterMessage,
+)
+from pydantic import ValidationError
+from websocket import WebSocket, WebSocketApp, WebSocketConnectionClosedException
+
+Message: TypeAlias = HelloMessage | NotificationMessage | RegisterMessage | UnregisterMessage
+Record: TypeAlias = HelloRecord | NotificationRecord | RegisterRecord
+
```

Comment:
Nope because it's global it should only be done in one place, especially since we invoke locust on both `locustfile/stored.py`. Those lines should probably move out of `locustfile.py` into a shared module eventually.